internal/compiler/passes/collect_structs_and_enums.rs

```diff
@@ -13,9 +13,9 @@ use std::rc::Rc;
 pub fn collect_structs_and_enums(doc: &Document) {
     let mut hash = BTreeMap::new();
 
-    for (name, exp) in doc.exports.iter() {
+    for (_, exp) in doc.exports.iter() {
```

Comment:
clippy would  tell you to use `.values()`

Reply:
Would that only work for a hash map? The exports are in a vector of tuples.